### PR TITLE
Export search publisher admin

### PR DESCRIPTION
**Pull request by vietdevne**
Publishers:
- Search by name
- Export list publisher to excel by query search

### CODE REVIEW

**Comment by vietdevne** [comment]
ready

File: resources/views/authors/view.blade.php [hunk 1]
```diff
@@ -13,37 +23,52 @@
...
       @foreach ($authors as $key => $author)
         <tr>
           <th scope="row">{{ $author->author_id }}</th>
           <td>{{ $author->author_name }}</td>
-          <td><img width="100" height="100" src="image/{{ $author->author_avatar }}" alt=""></td>
-          <td>{{ $author->author_desc }}</td>
           <td>
-            <a href="{{ route('admin.authors.edit', $author->author_id) }}" class="btn btn-warning">@lang('admin.actions.edit')</a>
+            @if ($author->author_avatar == null)
+              <img width="100" height="100" src="image/library.png" alt="">
```
Comment (quoctd-0607) [inline resources/views/authors/view.blade.php:43]: style viết ra file css

Comment (vietdevne) [reply resources/views/authors/view.blade.php:43]: dạ vâng ạ

**Comment by vietdevne** [comment]
ready

**Pull request merged**